src/action/config/ActionConfiguration.cpp

```diff
@@ -207,7 +207,7 @@ int ActionConfiguration::getUsedMeshID() const
       return mesh->getID();
     }
   }
-  PRECICE_ERROR("No mesh ID found!");
+  PRECICE_ERROR("No mesh ID found. Please check that the correct mesh name is used.");
```

Comment:
This really is internal knowledge and could confuse the user.

```suggestion
  PRECICE_ERROR("No mesh name \"" << _configuredAction.mesh << "\" found. Please check that the correct mesh name is used.");
```